kubernetes_deploy/production/ingress.yml

```diff
@@ -4,6 +4,13 @@ metadata:
   name: laa-cla-public
   namespace: laa-cla-public-production
 spec:
+  tls:
+  - hosts:
+    - laa-cla-public-production.apps.cloud-platform-live-0.k8s.integration.dsd.io
+  - hosts:
```

Comment:
To double check, this structure translates to
```json
{
  "spec": {
    "tls": [
      {
        "hosts": [
          "laa-cla-public-production.apps.cloud-platform-live-0.k8s.integration.dsd.io"
        ]
      },
      {
        "hosts": [
          "checklegalaid.service.gov.uk"
        ],
        "secretName": "laa-cla-public-production-tls-certificate"
      }
    ]
  }
}
```
in JSON. Is this the structure we want?